### ISSUE TITLE
Can you recommend some reference paper?

### ISSUE THREAD

**Issue by Ford666** [szuyul/MMF-simulation#5]
Hi, thanks so much for your shared code!
According to Cizmar's paper "Seeing through chaos in multimode fibres", where TM is measured in the representation of PIM, we know that the input focal spots are of orthogonal polarization states and also the outputs are acquired with two CCDs for both x-y polarized E fields. In your code of MMF_simTM_camera, I first get to know how to do the simulation! I see that there are some things involved like the H/V polarization excitation and the four quadrants of full transmission matrix, I wonder is there any reference paper for theoretical description? And when the dimension of output image is different from that of input, how to calculate mode_to_H/V (sim_M/cos_M can't be multiplied by Er/Ep) for H/V polarization of output?

**Comment by szuyul** [comment]
Hi, thanks for your interest in this topic.
1. transmission matrix is currently the most common approach to study light transport through a MMF. Basically, I think Cizmar's paper concisely explains the method. You may also want to study papers from Tomas Cizmar's, Joel Carpenter's, Demetri Psaltis', or Joseph Khan's groups. You can also take a look of our recent paper "Reciprocity-induced symmetry in the round-trip transmission through complex systems."
2. the input (N by N) and output (M by M) dimensions do not have to be same. In general, mode_to_H/V is a matrix where each column is a flattened M^2 by 1 image of a theoretical Bessel mode.
3. For the theory of MMF, I would recommend the textbook from Amnon Yariv "Optical Electronics For Modern Communications."

**Comment by Ford666** [comment]
@szuyul, thanks so much for your helpful reply for recommending the papers and textbook. 

For your point 2, mode_to_H/V actually describe horizontally/vertically polarized PIM of a MMF with given parameters. When M is different from N, is it the case that we should re-calculate the theoretical Er & Ep with image dimension M, so that we can multiply sin/cos_M of size (M^2,M^2) with Er/Ep of size (M^2, NMode) to obtain mode_to_H/V of size (M^2, NMode)?

**Comment by szuyul** [comment]
yes, I think projection into or out from modal basis could be done separately on the input/output sides. However, it would need re-calculation if M != N. So, to make things simpler, my example code make M = N. In general, you can adapt the code to different input/output dimensions.

**Comment by Ford666** [comment]
@szuyul, okay I see. Thanks a lot!
